content/helm-nginx/installnginx.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Install bitnami/nginx"
+date: 2018-08-07T08:30:11-07:00
+weight: 40
+---
+
+Installing the Bitnami standalone NGINX web server Chart involves us using the [helm install](https://docs.helm.sh/helm/#helm-install) command.
+
+When we install using Helm, we need to provide a deployment name, or a random one will be assigned to the deployment automatically.
+
+Let's provide **mywebserver** as the deployment name as we install NGINX:
+
+```
+helm install --name mywebserver bitnami/nginx
+```
+Upon running this command, you should see some confirmations around the objects that were created:
+
+```
+NAME:   mywebserver
+LAST DEPLOYED: Tue Nov 13 19:55:25 2018
+NAMESPACE: default
+STATUS: DEPLOYED
+
+RESOURCES:
+==> v1beta1/Deployment
```

Comment:
Can you talk about resources that are created? I can see this helm chart creates deployment, service etc.

Reply:
fixed